src/Model/Event.php

```diff
@@ -334,4 +334,25 @@ public static function hydrate(array $data, Calendar $calendar = null)
 
         return $event;
     }
+
+    public function export()
+    {
+        $export = [
+            'subject' => $this->getName(),
+            'bodyPreview' => $this->getDescription(),
+            'start' => ['dateTime' => $this->getStart()->format('c'), 'timeZone' => 'UTC'],
+            'end' => ['dateTime' => $this->getEnd()->format('c'), 'timeZone' => 'UTC'],
```

Comment:
careful here, on partial events `getStart` and `getEnd` doesn't _always_ return a date....